Aztec/Classes/Formatters/Implementations/MarkFormatter.swift

```diff
@@ -4,28 +4,43 @@ import UIKit
 class MarkFormatter: AttributeFormatter {
 
     var placeholderAttributes: [NSAttributedString.Key: Any]?
+    var textColor: String?
+    var defaulTextColor: UIColor?
```

Comment:
Optional nit:
```suggestion
    var defaultTextColor: UIColor?
```

Reply:
Nice catch! Updated in https://github.com/wordpress-mobile/AztecEditor-iOS/pull/1352/commits/7b9cc634d514835a0fd92d7743123211b06da4ee